Source/Train-Railway/FileManager.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Train_Railway
+{
+    class FilesPath
+    {
+        public static string PassengersPath { get; } = "Data/passengers.txt";
+        public static string TimetablePath { get; } = "Data/timetable.txt";
+        public static string ControllerPath { get; } = "Data/controllerlog.txt";
+        public static string StationPath { get; } = "Data/stations.txt";
+        public static string TrainPath { get; } = "Data/trains.txt";
+        public static string TrainTrackPath { get; } = "Data/traintrack.txt";
+    }
+
+
+    class FileManager
+    {
+        public static string[] ReadFile(string filePath)
+        {
+            string[] file = File.ReadAllLines(filePath);
+
+            return file;
+        }
+
+        public static void WriteFile(string filePath, string[] objToWrite)
+        {
+            File.WriteAllLines(filePath, objToWrite);
+        }
+        public static void WriteFile(string filePath, List<string> objToWrite)
+        {
+            File.WriteAllLines(filePath, objToWrite);
+        }
+
+        public static void SplitFile(string[] dataToSplit, List<Timetable> timetable)
+        {
+            for (int i = 1; i < dataToSplit.Length; i++)
+            {
+                string[] tmp = dataToSplit[i].Split(',');
+                timetable.Add(new Timetable(tmp[3], tmp[2], tmp[0], int.Parse(tmp[1])));
+            }
+        }
```

Comment:
You might be able to have one "SplitFile"-method instead and take a "List<Object> objects" in paramater instead of, in this case, "List<TimeTable> timetable". Then maybe add several if-statements in the method that checks the objects type (TimeTable, Passenger, Station etc).

**New method example:**  
`public static void SplitFile(string[] dataToSplit, List<Object> objects) { ... }`

**If-statement example**
`if (objects.GetType() == typeof(Timetable)) { ... }`